bigchaindb/processes.py

```diff
@@ -5,17 +5,16 @@
 from bigchaindb.pipelines import vote, block, election, stale
 from bigchaindb.web import server
 
-
 logger = logging.getLogger(__name__)
 
 BANNER = """
 ****************************************************************************
 *                                                                          *
 *   Initialization complete. BigchainDB is ready and waiting for events.   *
 *   You can send events through the API documented at:                     *
-*    - http://docs.bigchaindb.apiary.io/                                   *
+* https://bigchaindb.readthedocs.io/en/latest/drivers-clients/http-client-server-api.html *
```

Comment:
Note that this will have to change in regards to https://github.com/bigchaindb/bigchaindb/pull/650.